meta-iot2000-example/recipes-node/node-red/node-red_0.14.6.bb

```diff
@@ -0,0 +1,1974 @@
+# Recipe created by recipetool
```

Comment:
This boilerplate header should be replaced by something more appropriate after having checked the licenses etc.